zebra-chain/src/transaction/tests/vectors.rs

```diff
@@ -908,6 +908,36 @@ fn binding_signatures() {
                             )
                             .expect("a valid redjubjub::VerificationKey");
 
+                            bvk.verify(sighash.as_ref(), &sapling_shielded_data.binding_sig)
+                                .expect("verification passes");
+
+                            at_least_one_v5_checked = true;
+                        }
+                    }
+                    #[cfg(feature = "tx_v6")]
+                    Transaction::V6 {
+                        sapling_shielded_data,
+                        ..
+                    } => {
+                        if let Some(sapling_shielded_data) = sapling_shielded_data {
+                            // V5 txs have the outputs spent by their transparent inputs hashed into
```

Comment:
```suggestion
                            // V6 txs have the outputs spent by their transparent inputs hashed into
```

please let's use V6 everywhere possible.